### PR TITLE
sudachipy 0.6.7

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Python package https://github.com/WorksApplications/sudachi.rs/tree/develop/python
License from the sudashi.rs project https://github.com/WorksApplications/sudachi.rs/tree/develop

License: https://github.com/WorksApplications/sudachi.rs/blob/develop/LICENSE
Requirements:
- https://github.com/WorksApplications/sudachi.rs/blob/v0.6.7/python/pyproject.toml
- https://github.com/WorksApplications/sudachi.rs/blob/v0.6.7/python/setup.py

Actions:
1. Add the upstream license to the recipe folder
2. Skip [win64 and (rust_compiler == 'rust-gnu')], because `rust_compiler_rust-gnu` tries to create two artifacts with different hashes in the artifact's filename, and testing import of one of them fails but another succeeds. Another solution can be making a multi-output package for a gnu variant
3. Add $RPATH/ld64.so.1  to missing_dso_whitelist  on s390x 
4. Add -no-deps --no-build-isolation flags to script
5. Add rust compiler
6. Add {{ pin_compatible("libgcc-ng", max_pin=None) }} on linux
7. Add doc and dev urls
8. Add description
9. Add license_url
10. Add license_family

Notes:
- It's needed for resolving this issue https://github.com/conda-forge/spacy-models-feedstock/issues/2
- spacy data models waiting for sudachipy https://github.com/AnacondaRecipes/spacy-models-feedstock/pull/1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,37 +10,47 @@ source:
...
 about:
   home: https://github.com/WorksApplications/sudachi.rs/tree/develop/python
+  dev_url: https://github.com/WorksApplications/sudachi.rs/tree/develop/python
+  doc_url: https://github.com/WorksApplications/sudachi.rs/tree/develop/python
   summary: Python version of Sudachi, the Japanese Morphological Analyzer
+  description: Python version of Sudachi, the Japanese Morphological Analyzer
   license: Apache-2.0
-  license_file: PLEASE_ADD_LICENSE_FILE
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: why we need to add license to the recipe ? cannnot we add license_url pointing to https://github.com/WorksApplications/sudachi.rs/tree/develop/python  ?

File: recipe/LICENSE
Comment (JeanChristopheMorinPerso) [inline recipe/LICENSE:1]: Maybe you could use `license_url`?

**Review by JeanChristopheMorinPerso — COMMENTED** [review]
I'm not sure if we should embedded the license in the feedstock. I left a comment regarding that.

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: We can. But the license is placed in another folder https://github.com/WorksApplications/sudachi.rs/ that's why I've copied it to the recipe folder

File: recipe/LICENSE
Comment (skupr-anaconda) [reply recipe/LICENSE:1]: Yeah, we can

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

File: recipe/LICENSE
Comment (skupr-anaconda) [reply recipe/LICENSE:1]: Fixed

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**